artifactory/utils/pip/pip.go

```diff
@@ -45,6 +51,35 @@ func (pc *PipCmd) GetErrWriter() io.WriteCloser {
 	return pc.ErrWriter
 }
 
+func GetPipDepTreeScriptPath() (string, error) {
+	pipDependenciesPath, err := config.GetJfrogDependenciesPath()
+	if err != nil {
+		return "", err
+	}
+	pipDependenciesPath = filepath.Join(pipDependenciesPath, "pip", pipDepTreeVersion)
+
+	return writePipDepTreeScriptIfNeeded(pipDependenciesPath)
+}
+
+func writePipDepTreeScriptIfNeeded(targetPath string) (string, error) {
```

Comment:
Let's have this function do only what its name implies - write the file if needed. 
There's no need for it to also return the file.